src/tox_lsr/test_scripts/runcontainer.sh

```diff
@@ -115,6 +115,9 @@ refresh_test_container() {
              prepkgs="dnf-plugins-core" ;;
         *) pkgcmd=dnf; prepkgs="" ;;
         esac
+        for rpm in $EXTRA_RPMS; do
+            initpkgs="$rpm $initpkgs"
+        done
```

Comment:
I think it is ok to leave this, but in general, the test framework will install extra packages from tests/setup-snapshot.yml